src/main/java/com/google/api/generator/gapic/model/Field.java

```diff
@@ -73,6 +85,7 @@ public boolean equals(Object o) {
   @Override
   public int hashCode() {
     return 17 * name().hashCode()
+        + 57 * originalName().hashCode()
```

Comment:
not like it is of any importance, but why not 31? =)

Reply:
Done.